packages/@lightningjs/ui-components/src/utils/index.js

```diff
@@ -802,6 +802,25 @@ export function watchForUpdates({
   return element;
 }
 
+/**
+ * Given a Lightning text alignment option (left, right, center),
+ * return the Lightning flexbox equivalent.
+ *
+ * @param {string} align
+ * @returns {string}
+ */
+export function convertTextAlignToFlexJustify(align) {
+  switch (align) {
+    case 'left':
+      return 'flex-start';
+    case 'center':
+      return 'center';
+    case 'right':
+      return 'flex-end';
+  }
+  return 'flex-start'; // if there is no alignment passed in, the Lightning default is "left"
```

Comment:
is it possible use the switch statement `default` here instead of the return outside the switch?